src/organisms/cards/index.jsx

```diff
@@ -48,16 +54,12 @@ const PlantCard = ({
 					aria-expanded={expanded}
 					aria-label="show more"
 				>
-					<ExpandMoreIcon />
+					<Button size="small">Tell me More</Button> <ExpandMoreIcon />
 				</ExpandMore>
 			</CardActions>
 			<Collapse in={expanded} timeout="auto" unmountOnExit>
 				<CardContent>
-					<Typography variant="body1">Method:</Typography>
-					<Typography variant="body1">Lorem ipsum</Typography>
-					<Typography component="body1">Bliblablu</Typography>
-					<Typography component="body2">Foo Yoo Coo</Typography>
-					<Typography>Mimamu</Typography>
+					<Typography variant="body1">{tellMeMore} </Typography>
```

Comment:
According to MUI `body1` is the default value: https://mui.com/api/typography/

```suggestion
					<Typography>{tellMeMore} </Typography>
```